app/app.go

```diff
@@ -414,7 +422,7 @@ func NewEvmos(
 	// use custom Ethermint account for contracts
 	app.AccountKeeper = authkeeper.NewAccountKeeper(
 		appCodec, keys[authtypes.StoreKey],
-		evmostypes.ProtoAccount, maccPerms,
+		authtypes.ProtoBaseAccount, maccPerms,
```

Comment:
We're still using EthAccount type in this branch, no?
```suggestion
		evmostypes.ProtoAccount, maccPerms,
```

Reply:
Yeah just saw it was failing tests, must have been a mix up when porting the changes, nice catch @GAtom22!